### ISSUE TITLE
Store all files in a folder/possibility to use another folder insted of default Daz folder

### ISSUE THREAD

**Issue by Bercraft** [danielbui78/yaluxplug#3]
It would be great to choose the save folder for all the .ply files , .scn files etc.
Thank you.

**Comment by danielbui78** [comment]
Yes.  My plan is to add a separate "Export Files" option, so that people can easily do offline renders without Daz Studio.  It will allow you to specify folder to save out all necessary files.
